channels/us.m3u

```diff
@@ -711,14 +711,14 @@ https://otteravision-tg.ottera.tv/live/master.m3u8?channel=tg_tg_us
 https://cdn20.liveonlineservices.com/hls/eye95tv.m3u8
 #EXTINF:-1 tvg-id="FailArmy.us" tvg-name="Fail Army" tvg-country="US" tvg-language="English" tvg-logo="https://assets.tvplayer.com/tvos/images/channels/585.png" group-title="General",Fail Army (720p)
 https://dai.google.com/linear/hls/event/7tuuoX1wSsCTaki1HqJFYw/master.m3u8
-#EXTINF:-1 tvg-id="FailArmyXUMO.us" tvg-name="Fail Army (XUMO)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/RIXSg4L.png" group-title="",Fail Army (XUMO)
-https://dai2.xumo.com/xumocdn/p=redbox&deviceid=&is_lat=&subp=RedboxdesktopWebLinux/amagi_hls_data_xumo1212A-redboxfailarmy/CDN/playlist.m3u8
+#EXTINF:-1 tvg-id="FailArmyXUMO.us" tvg-name="Fail Army (XUMO)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/RIXSg4L.png" group-title="Comedy",Fail Army (XUMO)
+https://live-content.xumo.com/56/content/XM036MYDW1ETSM/22452084/4.m3u8
 #EXTINF:-1 tvg-id="FairfaxPublicAccess.us" tvg-name="Fairfax Public Access" tvg-country="US" tvg-language="English" tvg-logo="https://www.fcac.org/sites/all/themes/fcac_base/images/logo-icon.png" group-title="Legislative",Fairfax Public Access
 https://cs.ebmcdn.net/eastbay-live-hs-1/fairfax-pull/_definst_/mp4:fairfax.stream/chunklist.m3u8
 #EXTINF:-1 tvg-id="FaithLifeTV.us" tvg-name="Faith & Life TV" tvg-country="US" tvg-language="English" tvg-logo="https://pbs.twimg.com/profile_images/1131290789/fnlLogo_400x400.png" group-title="Religious",Faith & Life TV (720p)
 https://na-all9.secdn.net/logos-channel/live/faithlifetv/playlist.m3u8
-#EXTINF:-1 tvg-id="FamilyFeudXUMO.us" tvg-name="FamilyFeud (XUMO)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/uwBpm1Y.png" group-title="",FamilyFeud (XUMO)
-https://dai2.xumo.com/xumocdn/p=redbox&deviceid=&is_lat=&subp=RedboxdesktopWebLinux/amagi_hls_data_xumo1212A-redboxfamilyfeud/CDN/playlist.m3u8
+#EXTINF:-1 tvg-id="FamilyFeudXUMO.us" tvg-name="FamilyFeud (XUMO)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/uwBpm1Y.png" group-title="Entertainment",FamilyFeud (XUMO)
+https://live-content.xumo.com/170/content/XM00UVIHRILCPE/21427120/4.m3u8
```

Comment:
Isn't broadcast, returns a fixed chunk that ends

Reply:
Change it to playlist.m3u8 [https://cinedigm.vo.llnwd.net/conssui/amagi_hls_data_xumo1212A-redboxfamilyfeud/CDN/playlist.m3u8](url). It's because master.m3u8 on xumo wants to request an ad beacon.